### DISCUSSION TITLE
ORASENII

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#126] [TEAMs]
ECHIPA ORASENII se ocupa de orase.peviitor.ro

**Comment by AdrianaCionca** [comment]
Adelina Guliman
role: Team Lead & Test Design
discord: Adelina

**Comment by AdrianaCionca** [comment]
Florin Nistor
role: Test Design
discord: Florin

**Comment by AdrianaCionca** [comment]
Adina Boeru
role: Test Run
discord: Adina Boeru

**Comment by AdrianaCionca** [comment]
Bogdan Dumitrache
role: Test Design
discord: Bogdan D.

**Comment by AdrianaCionca** [comment]
Andra Adamache
role: Test Run
discord: Andra

**Comment by AdrianaCionca** [comment]
Beatrix Racasanu
role: Test Design
discord: Beatrix

**Comment by AdrianaCionca** [comment]
Diana Dragoi
role: Test Design
discord: deedee_diana

**Comment by AdrianaCionca** [comment]
Simona Nemtanu
role: Test Design
discord: Simona Nemtanu

**Comment by AdrianaCionca** [comment]
Romeo Chiparca
role: Test Design
discord: Romeo81

**Comment by AdrianaCionca** [comment]
Cristina Voicu
role: Test Run
discord: Cristina Voicu
